packages/rs-drive-nodejs/src/lib.rs

```diff
@@ -2243,26 +2274,42 @@ impl PlatformWrapper {
                 Ok(None)
             };
 
-            let mut batch = GroveDbOpBatch::new();
+            let block_info = drive::drive::block_info::BlockInfo {
+                time_ms: 1,
+                height: 1,
+                epoch: Epoch::new(1),
+            };
+
+            let mut drive_operations = vec![];
 
             let index_bytes = (index as u64).to_be_bytes().to_vec();
 
-            let withdrawals = vec![(index_bytes.clone(), transaction_bytes)];
+            let withdrawals = vec![(index_bytes, transaction_bytes)];
 
-            platform
-                .drive
-                .add_enqueue_withdrawal_transaction_operations(&mut batch, withdrawals);
+            let execution_function = |transaction: TransactionArg| -> Result<(), String> {
```

Comment:
You probably need a method: `Drive.enqueue_withdrawal_transaction` with apply batch inside.